src/Makefile

```
@@ -24,7 +24,7 @@ MAC_LDFLAGS=-bundle -undefined dynamic_lookup $(LIBDIR)
 
 INSTALL  = install
 CC      ?= cc
-LD      ?= $(MYENV) cc
+LD      ?= $(MYENV) $(CC)
```

```suggestion
CCLD      ?= $(MYENV) $(CC)
```
If you keep setting LD, it will get overridden to `$(CROSSPREFIX)-ld` when cross-compiling all the same.  It should be renamed.  `CCLD` would be a good choice to express what it really is: invocation of $CC to do the linking.
Then line 54 would be changed to:
```
	$(CCLD)  $(LDFLAGS) -o $@ $(OBJS) $(LIBS)
```

---

> If you keep setting LD, it will get overridden to $(CROSSPREFIX)-ld when cross-compiling all the same.

That's rather the whole point of overriding LD, isn't it? What's the rationale behind breaking that?